### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?180879#1012374

http://manage.dimagi.com/default.asp?180880#1012373

@TylerSheffels

### CODE REVIEW

**Comment by TylerSheffels** [comment]
@emord Looks good to me, but I have 0 vellum context

File: src/logic.js [hunk 1]
```diff
@@ -206,6 +206,7 @@ define([
                 if (!refMug &&
                     (!mug.options.ignoreReferenceWarning || !mug.options.ignoreReferenceWarning(mug)) &&
                     _this.opts.allowedDataNodeReferences.indexOf(pathWithoutRoot) === -1 &&
+                    !pathWithoutRoot.startsWith('meta/') &&
```
Comment (orangejenny) [inline src/logic.js]: Worth updating https://github.com/dimagi/Vellum/blob/master/tests/options.js#L93-L98 ?  Is allowedDataNodeReferences now ever used for paths other than `/data/meta/*`?

Comment (orangejenny) [inline src/logic.js]: (not blocking)

File: tests/logic.js [hunk 2]
```diff
@@ -113,6 +113,14 @@ require([
                 });
             });
         });
+
+        describe("should not add validation error for", function() {
+            it("meta reference", function () {
```
Comment (orangejenny) [inline tests/logic.js]: How is this testing meta references? A form with a single text question doesn't have any paths with meta in them (I thought...).

Comment (emord) [inline tests/logic.js]: d'oh somehow missed that line during commit. updated

**Pull request merged**